paddle/fluid/framework/new_executor/interpretercore.cc

```diff
@@ -786,16 +790,25 @@ void InterpreterCore::ExecuteInstructionList(
     return;
   }
 
+  std::unique_ptr<std::vector<std::atomic<size_t>>> atomic_deps = nullptr;
```

Comment:
Although it may be negligible, the time cost of constructing and initializing atomic_deps and atomic_var_ref should be counted in record_prepare Guard.

Reply:
done